python/ray/__init__.py

```diff
@@ -46,6 +46,9 @@
             e.args += (helpful_message, )
     raise
 
+modin_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), "modin")
```

Comment:
I added a warning. I am not sure that we should throw an error because we don't know the intention of the user in this case. What do you think?

Reply:
I'd be in favor of throwing an error and then reconsidering if it's an issue, but warnings are easy to ignore, and I'd prefer to throw an interpretable error here rather than some opaque version issue later.

Reply:
Great! added it as an `ImportError`

Reply:
Quick note to mention that doing this creates issues in Modin when we import ray. We can resolve them in Modin at a later date, but for now we cannot throw an error.